draft-zip.md

```diff
@@ -0,0 +1,97 @@
+```
+ZIP: 
+Title: Deposit 50% of transaction fees to the Zcash Sustainability Fund
+Owners: Jason McGee <jason@shieldedlabs.com>
+        Mark Henderson <mark@equilibrium.co>
+        Tomek Piotrowski <tomek@eiger.co>
+        Mariusz Pilarek <mariusz@eiger.co>
+Original-Authors: Nathan Wilcox
+Credits: Nathan Wilcox
+         Mark Henderson
+         Jason McGee
+         Tomek Piotrowski
+         Mariusz Pilarek
+Status: Draft
+Category: Ecosystem
+Created: 2023-09-
```

Comment:
Fixed but left the date. Should I just us an arbitrary date (today-ish) for the Created date?

Reply:
I had to look this up, because I wasn't sure:
> The Created header records the date that the ZIP was submitted. Dates should be in yyyy-mm-dd format, e.g. 2001-08-14.

https://zips.z.cash/zip-0000#zip-header-preamble